File: sprint1/Java/lesson6code1.java
int[] cumulativeSums = new int[sequence.length() + 1];
int cgCount = 0;
for (int position = 0; position < sequence.length(); position++) {
    if (sequence.charAt(position) == 'C' || sequence.charAt(position) == 'G') {
        cgCount++;
    }
    cumulativeSums[position + 1] = cgCount;
}
System.out.println(cumulativeSums[right] - cumulativeSums[left]);

File: sprint1/C++/lesson6code1.cpp
#include <iostream>
using namespace std;

int main() {
    int cg_count = 0;
    int left = 0; // replace with actual value
    int right = 10; // replace with actual value
    string sequence = "ATCGCTAGCG"; // replace with actual value

    for (int position = left; position < right; position++) {
        if (sequence[position] == 'C' || sequence[position] == 'G') {
            cg_count++;
        }
    }

    cout << cg_count << endl;

    return 0;
}
